application/core/LSETwigViewRenderer.php

```
@@ -730,6 +731,37 @@ private function getAdditionalInfos($aData, $oTemplate)
+        if ($DefaultPrivacyDone) {
+            return $aSurveyInfo;
+        }
+        $DefaultPrivacyDone = true;
+        if (empty($aSurveyInfo['datasecurity_notice_label'])) {
+            $aSurveyInfo['datasecurity_notice_label'] = gT("To continue please first accept our survey privacy policy.");
+        }
+        if (empty($aSurveyInfo['datasecurity_error'])) {
+            $aSurveyInfo['datasecurity_error'] = gT("We are sorry but you can't proceed without first agreeing to our survey privacy policy.");
+        }
+        /* @var string[] to go to automatic translation */
+        $translation = [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$translation'. (undefined)

(UnusedLocalVariable)